x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/action/interceptor/ResizeRequestInterceptor.java

```diff
@@ -59,7 +59,7 @@ public void intercept(ResizeRequest request, Authentication authentication, Role
                 userPermissions.indices().allowedActionsMatcher(request.getTargetIndexRequest().index());
             if (Operations.subsetOf(targetIndexPermissions, sourceIndexPermissions) == false) {
                 // TODO we've already audited a access granted event so this is going to look ugly
-                auditTrailService.accessDenied(authentication, action, request, userPermissions.names());
+                auditTrailService.accessDenied(null, authentication, action, request, userPermissions.names());
```

Comment:
idem:

I prefer us to be explicit here: threadContext.getTransient(AUDIT_REQUEST_ID).
It is a case of tamper request , if audit request Id is missing.

Reply:
As above - but let's chat if you disagree.

Reply:
I've clarified my thoughts in the previous comment. Judging from the code flow, I believe we can be explicit about the id and not pass `null`.